### PR TITLE
Run and store ldbc benchmarks

### PR DESCRIPTION
**Pull request by gz**

### CODE REVIEW

File: benches/ldbc-graphalytics/main.rs [hunk 1]
```diff
@@ -43,6 +45,9 @@ use std::{
 #[global_allocator]
 static ALLOC: MiMalloc = MiMalloc;
 
+// If you change this, also adjust `scripts/ci.bash`:
+const LDBC_RESULT_CSV: &str = "ldbc_results.csv";
```
Comment (Kixiron) [inline benches/ldbc-graphalytics/main.rs:49]: Maybe this should be a cli parameter?

File: benches/ldbc-graphalytics/main.rs [hunk 2]
```diff
@@ -238,6 +243,74 @@ fn main() {
...
+                // If the file exists, we append another row, this is different
+                // from other benchmarks that can run everything with a single
+                // invocation:
+                let file = OpenOptions::new()
+                    .write(true)
+                    .append(results_file_already_exists)
+                    .create(!results_file_already_exists)
+                    .open(LDBC_RESULT_CSV)
+                    .expect("failed to open results csv file for writing");
+                    let mut csv_writer = csv::Writer::from_writer(file);
```
Comment (Kixiron) [inline benches/ldbc-graphalytics/main.rs:257]:
```suggestion
                let mut csv_writer = csv::Writer::from_writer(file);
```

**Review by Kixiron — APPROVED** [review]

File: benches/ldbc-graphalytics/main.rs (hunk 2 above)
Comment (gz) [reply benches/ldbc-graphalytics/main.rs:257]: It's weird that cargo fmt didn't format this (or complain about it) -- does it not check things in benches?

Comment (gz) [reply benches/ldbc-graphalytics/main.rs]: (I did run `cargo fmt` and noticed it didn't touch/fix formatting issue in this file)

Comment (gz) [reply benches/ldbc-graphalytics/main.rs]: (Updating cargo fmt seems to have helped its formatting again)

File: benches/ldbc-graphalytics/main.rs (hunk 1 above)
Comment (gz) [reply benches/ldbc-graphalytics/main.rs]: Fixed

**Pull request merged**